### PR TITLE
Javascript/javascript3/week2

### PR DESCRIPTION
**Pull request by Bhuvaneshwari29**
JS3 week2 homework completed.

### CODE REVIEW

File: javascript/javascript3/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+//Movies exercise
+const movieUrl =
+  "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json";
+async function movieFunction() {
```
Comment (senner007) [inline javascript/javascript3/week2/index.js:4]: Try and seperate the api call from the data manipulation

File: javascript/javascript3/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      console.log(
+        "for using promises and .then : astro number are ",
+        data.number
+      )
+    );
+});
+//Do the 3 steps below using async/await
+async function useAsyncAwait() {
+  try {
+    await usePromise;
```
Comment (senner007) [inline javascript/javascript3/week2/index.js:85]: Here you are fetching twice right?

File: javascript/javascript3/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    await moveElement(greenColorBox, greenCircle);
+    console.log("Green box moved");
+  } catch (error) {
+    console.log(error.message);
+  }
+}
+translateOneByOne();
+
+//translate all at once
+function translateAllAtOnce() {
```
Comment (senner007) [inline javascript/javascript3/week2/main.js:35]: I think the moveElement functions should go inside the Promise.all as parameters
 Promise.all([  moveElement(redColorBox, redCircle)
, moveElement(blueColorBox, blueCircle), moveElement(greenColorBox, greenCircle)])

**Review by senner007 — APPROVED** [review]
Good :)
See the comments

File: javascript/javascript3/week2/index.js (hunk 1 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week2/index.js:3]: Api call separated from the data manipulation

File: javascript/javascript3/week2/index.js (hunk 2 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week2/index.js]: Oops..Yes deleted that

File: javascript/javascript3/week2/main.js (hunk 3 above)
Comment (Bhuvaneshwari29) [reply javascript/javascript3/week2/main.js:35]: Yes..moved it.

**Pull request merged**